client/blocks/login/two-factor-authentication/two-factor-actions.jsx

```diff
@@ -37,15 +35,21 @@ class TwoFactorActions extends Component {
 		page( login( { twoFactorAuthType: 'sms' } ) );
 
 		this.props.sendSmsCode( userId, twoStepNonce ).then( ( phoneNumber ) => {
-			this.props.successNotice(
-				translate( 'A text message with the verification code was just sent to your phone number ending in %(phoneNumber)s', {
-					args: {
-						phoneNumber: phoneNumber
+			this.props.setNotice( {
+				message: translate( 'A text message with the verification code was just sent to your ' +
+					'phone number ending in %(phoneNumber)s', {
+						args: {
+							phoneNumber: phoneNumber
+						}
 					}
-				} )
-			);
+				),
+				status: 'is-success'
```

Comment:
These are actually properties for the `Notice` component, not CSS classes. Happy to translate it if you think its worth the extra code.